great_expectations/rule_based_profiler/expectation_configuration_builder/default_expectation_configuration_builder.py

```diff
@@ -349,7 +349,7 @@ def _build_expectation_configuration(
         variables: Optional[ParameterContainer] = None,
         parameters: Optional[Dict[str, ParameterContainer]] = None,
         runtime_configuration: Optional[dict] = None,
-    ) -> Optional[ExpectationConfiguration]:
+    ) -> ExpectationConfiguration | None:
```

Comment:
Just curious: was this change necessary, or just a style thing?

Reply:
just a style thing